nemo_aligner/data/nlp/builders.py

```diff
@@ -280,9 +280,6 @@ def build_sft_dataset(data_cfg, tokenizer, num_samples, answer_only_loss=True, i
         index_mapping_dir=data_cfg.get("index_mapping_dir", None),
         prompt_template=data_cfg.get("prompt_template", None),
         virtual_tokens=0,
-        tokens_to_generate=data_cfg.get(
```

Comment:
just so i understand: because we generate and pad our own data, we don't need this argument?

Reply:
it was unused before and in the current code this comes from the length parameters of the model.infer()